src/dispensary/updater.js

```diff
@@ -54,7 +42,8 @@ export default class Updater {
         return this.outputHashes(libraries);
       })
       .then((hashes) => {
-        return this[`${this.command}Command`](hashes, _console);
+        _console.log(hashes.join('\n'));
+        return Promise.resolve(hashes);
```

Comment:
Shouldn't we have the logic of the former `updateCommand` here? If I understand correctly, this would not update the hashes file?

Reply:
The docs I ported to the README indicates that we have to redirect the output of the command to the txt file, so I kept that. It is useful because that way we kinda have two options: showing the content of the hashes and updating the file (by redirecting the output).

We can add a npm script to abstract this part so that `npm run update-hashes` updates the txt file directly.

Reply:
Ah! Yes please on having an npm script. Can we add that to this PR or file a follow-up so we don't forget?

Reply:
added in this patch

Reply:
Sorry I missed that. Thanks!